### ISSUE TITLE
`Cannot use import statement outside a module` when using a TypeScript config file

### ISSUE THREAD

**Issue by michaelschufi** [drizzle-team/drizzle-kit-mirror#44]
Hi

I'm running into the problem that I cannot use the TypeScript config file, e.g.  `drizzle.config.ts`.

Note: I'm using the `@db-push` tag.

My config file:
```ts
import 'dotenv/config';

import type { Config } from 'drizzle-kit';

import { env } from '@/env.mjs';

export default {
  schema: './src/backend/db/schema',
  out: './src/backend/db/migrations',
  host: env.DATABASE_HOST,
  user: env.DATABASE_USERNAME,
  password: env.DATABASE_PASSWORD,
} satisfies Config;
```

My terminal output:

`pnpm exec drizzle-kit push:mysql`
```
drizzle-kit: v0.17.6
drizzle-orm: v0.25.4

No config path provided, using default 'drizzle.config.ts'
Reading config file '/path-to-monorepo/project-folder/app/drizzle.config.ts'
/path-to-monorepo/project-folder/app/src/env.mjs:2
import { createEnv } from "@t3-oss/env-nextjs";
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1048:15)
    at Module._compile (node:internal/modules/cjs/loader:1083:27)
    at Module._compile (/path-to-monorepo/node_modules/.pnpm/drizzle-kit@0.17.6-76e73f3/node_modules/drizzle-kit/index.js:16330:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1173:10)
    at Object.newLoader [as .mjs] (/path-to-monorepo/node_modules/.pnpm/drizzle-kit@0.17.6-76e73f3/node_modules/drizzle-kit/index.js:16334:13)
    at Module.load (node:internal/modules/cjs/loader:997:32)
    at Module._load (node:internal/modules/cjs/loader:838:12)
    at Module.require (node:internal/modules/cjs/loader:1021:19)
    at require (node:internal/modules/cjs/helpers:103:18)

Node.js v18.9.1
```

Last but not least, my tsconfig:
```json
{
  "ts-node": {
    "moduleTypes": {
      "src/backend/db": "esm"
    },
    "cwd": "/path-to-monorepo/project-folder/app",
    "projectSearchDir": "/path-to-monorepo/project-folder/app",
    "project": "/path-to-monorepo/project-folder/app/tsconfig.json"
  },
  "compilerOptions": {
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "module": "esnext",
    "moduleResolution": "node",
    "jsx": "preserve",
    "strict": true,
    "allowJs": true,
    "skipLibCheck": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": false,
    "esModuleInterop": true,
    "resolveJsonModule": true,
    "isolatedModules": true,
    "useDefineForClassFields": true,
    "incremental": true,
    "baseUrl": "./",
    "paths": {
      "@/*": [
        "./src/*"
      ]
    },
    "target": "es6",
    "sourceMap": true,
    "inlineSourceMap": false,
    "inlineSources": true,
    "declaration": false,
    "outDir": "./.ts-node"
  }
}
```

What can I do to make this work?

**Comment by mikebuilds** [comment]
Did you get this one working?

**Comment by michaelschufi** [comment]
Nope, I just removed the `env.mjs` dependency. Which is a shame, but the impact to maintainability is not that high and the same variables are needed elsewhere in the project, so they still get validated.

**Comment by AndriiSherman** [comment]
Should be fixed in 0.19.0
